src/fs.h

```diff
@@ -32,9 +32,9 @@ class path : public std::filesystem::path
     using std::filesystem::path::path;
 
     // Allow path objects arguments for compatibility.
-    path(std::filesystem::path path) : std::filesystem::path::path(std::move(path)) {}
-    path& operator=(std::filesystem::path path) { std::filesystem::path::operator=(std::move(path)); return *this; }
-    path& operator/=(std::filesystem::path path) { std::filesystem::path::operator/=(std::move(path)); return *this; }
+    path(std::filesystem::path path) : std::filesystem::path::path{path} {}
+    path& operator=(std::filesystem::path path) { std::filesystem::path::operator=(path); return *this; }
+    path& operator/=(std::filesystem::path path) { std::filesystem::path::operator/=(path); return *this; }
```

Comment:
In commit "Remove std::move from fs wrapper to work around bug" (faf7a61483a959c1a45d5bf7d304809f506a2ce6)

If you want to avoid more moves here, you should change the `std::filesystem::path path)` arguments to `const std::filesystem::path path`). So for example if someone writes

```c++
path1 = path2 / path3;
```

Current code moves from the (`path2 / path3`) temporary, and commit faf7a61483a959c1a45d5bf7d304809f506a2ce6 still moves from that temporary. With const reference operator=, the temporary can be copied from instead of moved from, which seems like what we would want to be safe from the bug